### ISSUE TITLE
Impossible to format pug with 'each'

### ISSUE THREAD

**Issue by Applelo** [prettier/plugin-pug#29]
I use the "Format Document" option with the VSCode extension on this simple pug file:

```
- const files = ['script.js']

each jsFile in files
  script(src=jsFile)
```

But I got this error.

`
Unhandled token/Users/loisboubault/Tools/test/test.pug:: {"type"/Users/loisboubault/Tools/test/test.pug::"each","loc"/Users/loisboubault/Tools/test/test.pug::{"start"/Users/loisboubault/Tools/test/test.pug::{"line"/Users/loisboubault/Tools/test/test.pug::3,"column"/Users/loisboubault/Tools/test/test.pug::1},"end"/Users/loisboubault/Tools/test/test.pug::{"line"/Users/loisboubault/Tools/test/test.pug::3,"column"/Users/loisboubault/Tools/test/test.pug::21}},"val"/Users/loisboubault/Tools/test/test.pug::"jsFile","key"/Users/loisboubault/Tools/test/test.pug::null,"code"/Users/loisboubault/Tools/test/test.pug::"files"}
`
Do prettier pug plugin supports "each" iterations ?

**Comment by Shinigami92** [comment]
In a few days it will be supported 😉

**Comment by Shinigami92** [comment]
Turns out that was very easy to implement... a little bit to easy.
There is a `key` in the `each`-Token and the value of this `key` is `null` in your provided example.
So in this current implementation it could be buggy for special cases. If this turns out to be a problem, please report an additional Issue with logs!  Thx 🙏

**Comment by Shinigami92** [comment]
Released as [1.0.6](https://www.npmjs.com/package/@prettier/plugin-pug/v/1.0.6)
Thanks for using my Plugin 😃
